BonillaDABI/DurandalTest
# Recurrent visit front, WORKING, PENDING & READY in desc

WORKING:
* Fixing toggle buttons in ModalRecurrVisit
* Button permissions
	* Import excel permission
	* Export excel permission
	* Multiple delete permission (same as simple delete)

READY:
+ Recurrent visit frontend implementation
   + Checkbox
   + Select
   + ModalRecurrVisit for 'Personalizado...'
+ Recovery password <a> to skip request code
+ Folder for all CREATE Modals

+ Test recovery password
   + Send email
   + Validate code
   + Change to new password
+ Axios for recovery password
   + Send email
   + Validate code
   + Change to new password (send new password and code)
+ Test visits
+ Recovery screen
+ Future visits radio button [programada] disabled when date is today
+ Error code in no perm page

PENDING:
- Recurrent visit functionality implementation
- Check password possible cases after code validated.
- Buttons permissions
